### PR TITLE
[Translation zh-cn] Resources > Translations.md

### PR DESCRIPTION
**Pull request by limhenry**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-CN/review/377)

### CODE REVIEW

File: src/content/zh-cn/resources/translations.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+3. 寻找您想要翻译的文章。
+4. 开始翻译。
+5. 当翻译完毕，放送您的翻译以进行检查。
+6. 检查完毕的文章将会被语言管理员作为Pull Request发送到GitHub。
+
+您可以从GitLocalize的[帮助页面](https://docs.gitlocalize.com/)学到GitLocalize是如何操作的。
+
+如果您遇到任何问题或者有功能请求，您可以在[GitLocalize的issue tracker](https://github.com/gitlocalize/feedback/issues)提供建议。
+
+與其他翻譯貢獻者聊天，您可以[註冊ChromiumDev的Slack](https://chromiumdev-slack.herokuapp.com/)並加入#l10n頻道。
```
Comment (petele) [inline src/content/zh-cn/resources/translations.md]: `Note:` formatting

**Comment by petele** [comment]
this feels like it's missing a lot of content from the current version

**Comment by limhenry** [comment]
actually this is not synced with the latest version (same as #5332)

**Comment by limhenry** [comment]
Updated version: #5607

**Pull request merged**